From b91a6f3618f8397f9aa370b1288dc9336277bed7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Sat, 24 Jul 2010 18:04:30 +0000
Subject: [PATCH v2 1/7] gitignore: Ignore files generated by "make coverage"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The "make coverage" support added by Thomas Rast in 901c369af5 didn't
contain a corresponding patch to patch .gitignore.

Change gitignore to ignore the *.gcda, *.gcno and *.gcov files
generated by GCC and our coverage invocations.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 .gitignore |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/.gitignore b/.gitignore
index 14e2b6b..0e61845 100644
--- a/.gitignore
+++ b/.gitignore
@@ -204,3 +204,7 @@
 *.pdb
 /Debug/
 /Release/
+*.gcda
+*.gcno
+*.gcov
+/coverage-untested-functions
-- 
1.7.0.4

